internal/layout/timeline.go

```diff
@@ -103,6 +105,8 @@ func LayoutTimeline(layout Layout, collection collection.Collection, scene *rend
 		Interval: 1 * time.Second,
 	}
 
+	locations := make(map[string]bool)

```

Comment:
The dedup here is pretty good already, but it can still be a bit verbose at times. One idea might also be to only list the different cities if they are all in the same country, or different countries or regions if not. That might take a bit to figure out though, maybe better left for v2 :)